From b6ea96c22034d2dc4ea0f7adba1f6f48e79d9e13 Mon Sep 17 00:00:00 2001
From: Arnaud Vrac <avrac@freebox.fr>
Date: Sun, 10 May 2015 20:10:49 +0200
Subject: [PATCH 2/2] mpegtsdemux: do not abort playback when no PCR were found
 during initial scan

Seeking will not be possible in those streams but at least the can be
played. Note scanning is only done when tsdemux is configured in pull mode.
---
 gst/mpegtsdemux/mpegtsbase.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/mpegtsdemux/mpegtsbase.c b/gst/mpegtsdemux/mpegtsbase.c
index 0ca7253..e6c5fde 100644
--- a/gst/mpegtsdemux/mpegtsbase.c
+++ b/gst/mpegtsdemux/mpegtsbase.c
@@ -1310,7 +1310,7 @@ no_initial_pcr:
   mpegts_packetizer_clear (base->packetizer);
   GST_WARNING_OBJECT (base, "Couldn't find any PCR within the first %d bytes",
       10 * 65536);
-  return GST_FLOW_ERROR;
+  return GST_FLOW_OK;
 }
 
 
-- 
1.9.5 (Apple Git-50.3)
